docs/content/stable/yugabyte-voyager/reference/configuration-file.md

```diff
@@ -112,6 +119,12 @@ source:
   # Can be specified via SOURCE_DB_PASSWORD environmental variable
   db-password: test_password
 
+  # Comma-separated list of read replica endpoints.
+  # Default port: 5432
+  # Example: "host1:5432, host2:5433"
```

Comment:
- unclear whether quotes are required. Apparently not but this implies the opposite I would ditch the quotes
- does this mean including the port number is optional?